test/lib/attributes.js

```diff
@@ -18,7 +18,7 @@ describe('lib/attributes', () => {
     assert.isTrue(!!list.id);
     assert.equal(input.getAttribute('role'), 'combobox');
     assert.equal(list.getAttribute('role'), 'listbox');
-    assert.equal(input.getAttribute('aria-owns'), list.id);
+    assert.equal(input.getAttribute('aria-controls'), list.id);
```

Comment:
This line of  code is not updated in previous PR. Hence, changes accordingly with recent unit test fixes.